src/shared/ui/image/utils/checkForSuitable.ts

```diff
@@ -0,0 +1,19 @@
+const checkForSuitable = (imgUrl: string, width: number = 430) => {
+  const extention = imgUrl.slice(imgUrl.lastIndexOf('.') + 1);
+
+  if (extention === 'svg') {
+    return true;
+  }
+
+  const image = new Image();
+
+  image.src = imgUrl;
+
+  if (image.width <= width) {
+    return true;
+  }
+
+  return false;
+};
+
+export default checkForSuitable;
```

Comment:
Please, remove default exports.

Reply:
The current implementation of utilities is done  with default export. I wouldn't want to change this PR